src/js/dropdown.ts

```diff
@@ -1,20 +1,51 @@
 import Choices from "choices.js";
 import "choices.js/public/assets/styles/choices.css";
 import scoreCardsData from "../../data/score-cards.json";
-import { CityId } from "./types";
+import { CityId, ScoreCardDetails } from "./types";
 
 export const DROPDOWN = new Choices("#city-choice", {
   allowHTML: false,
   itemSelectText: "Select",
   searchEnabled: true,
+  shouldSort: false, // since cities are already alphabetical order in scorecard, disabling this option allows us to show PRN maps at the top.
 });
 
+const typedScoreCardData: Record<string, ScoreCardDetails> = scoreCardsData;
+
 const setUpDropdown = (initialCityId: CityId, fallBackCityId: CityId) => {
-  const cities = Object.entries(scoreCardsData).map(([id, { name }]) => ({
-    value: id,
-    label: name,
-  }));
-  DROPDOWN.setChoices(cities);
+  const allCities = Object.entries(typedScoreCardData).map(
+    ([id, { name, contribution }]) => ({
+      value: id,
+      label: name,
+      contribution: contribution || "PRN",
+    })
+  );
+  const communityCities = allCities.filter(
+    (city) => city.contribution !== "PRN"
+  );
```

Comment:
It would be more efficient to create `communityCities` and `officialCities` in one go so that you only iterate one time.

```typescript
const officialCities = []
const communityCities = []
Object.entries(scoreCardsData as Record<string, ScoreCardDetail>).forEach(([id, { name, contribution }]) => {
   const entry = {
      value: id,
      label: name,
      contribution: contribution || "PRN",
    }
    if (contribution) {
      communityCities.push(entry)
    else {
      officialCities.push(entry)
    }
})